core/data/src/main/java/team/ppac/data/mapper/MemeResponceMapper.kt

```diff
@@ -0,0 +1,19 @@
+package team.ppac.data.mapper
+
+import team.ppac.domain.model.Meme
+import team.ppac.remote.model.response.meme.MemeResponse
+
+internal fun MemeResponse.toMeme(): Meme {
+    return Meme(
+        id= id,
+        image= image,
+        isTodayMeme= isTodayMeme,
+        source= source,
+        title= title,
+        watch= watch,
+        reaction= reaction,
+        createdAt= createdAt,
+        updateAt= updatedAt,
+        keywords= keywords,
+    )
+}
```

Comment:
```suggestion
internal fun MemeResponse.toMeme(): Meme = Meme(
    id= id,
    image= image,
    isTodayMeme= isTodayMeme,
    source= source,
    title= title,
    watch= watch,
    reaction= reaction,
    createdAt= createdAt,
    updateAt= updatedAt,
    keywords= keywords,
)
```
바로 리턴 조져버리는거면 요렇게는 오또신쥐

Reply:
리턴축약 메모메모~ 반영했습니당